go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/golang/mock v1.6.0 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/golang/snappy v0.0.4 // indirect
+	github.com/google/btree v1.1.2 // indirect
```

Comment:
You should update avalanchego version to the latest 1.9.0 which includes relevant changes.
Also note that previous Blueberry name has been changed to Banff. We removed Blueberry from avalanchego codebase, you should do it too here.

Reply:
So mainnet is under deployment today (see release [here](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0)), so it should be safe to merge, unless your workflow has reasons I am not aware of.

Reply:
done